conda-recipes/cyclus-deps/post-link.sh

```diff
@@ -1,30 +1,3 @@
-#/bin/bash
```

Comment:
I was wondering about the whole file being deleted but just tagged it on this line.